### PR TITLE
Feat: alert, avatar, button and badge

### PR DESCRIPTION
**Pull request by Ashutosh-Bhadauriya**
Added alert, badge, avatar and button component

### CODE REVIEW

**Comment by SomeEarth86** [comment]
PR description must contains details about files.

File: components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  font-size: 1.6rem;
+  margin: 1rem 0;
+  position: relative;
+  width: 100%;
+  box-sizing: border-box;
+}
+.fa {
+  margin-right: 0.625rem;
+  width: 1em;
+  height: 1em;
```
Comment (SomeEarth86) [inline components/alert/alert.css:13]: Try to use one unit throughout the code, be it `rem` or `em`.

File: components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+.alert-primary {
+  background-color: #eaf1ff;
+  color: var(--primary-color);
+}
+.alert-box .link-text,
+.alert-box .btn {
+  font-weight: 500;
+}
+.alert-secondary {
+  background-color: #fff3d9;
```
Comment (SomeEarth86) [inline components/alert/alert.css:25]: use of CSS variable should be maximised.

Comment (SomeEarth86) [inline components/alert/alert.css:25]: use of CSS variable should be maximised.

**Review by INNOMIGHT — APPROVED** [review]
No conflicts or issues.

**Pull request merged**